base/docs/Docs.jl

```diff
@@ -395,9 +395,18 @@ function summarize(io::IO, T::DataType, binding)
     if !isempty(subtypes(T))
         println(io, "**Subtypes:**")
         println(io, "```")
-        for t in subtypes(T)
+        sub_types = subtypes(T)
+        for t in Iterators.take(sub_types, 20)
```

Comment:
do we hardcode limits like this elsewhere? could maybe be an iocontext property eventually